.travis.yml

```diff
@@ -17,6 +17,10 @@ matrix:
   #     we can build on an older SDK.
   - dart: stable
     env: BOT=main
+  # TODO(dantup): Temporarily skip stable DDC builds until the interop fix is
```

Comment:
add a bug # for this so we get reminded to uncomment the tests.

Reply:
I've opened https://github.com/flutter/devtools/issues/748 and linked it back here.